pkgs/development/python-modules/matrix-client/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, pythonPackages
+}:
+
+buildPythonPackage rec {
+  pname = "matrix-client";
+  version = "0.0.6";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "15kx5px26hwr0sxpyjk4w61fjnabg1b57hwys1nyarc0jx4qjhiq";
+  };
+
+  buildInputs = with pythonPackages; [ requests ];
+
+  # Tests need extra requirements, disabling for now.
+  doCheck = false;
```

Comment:
Note that this package may break anytime without knowing if there aren't any tests.